nodebuilder/node.go

```diff
@@ -56,6 +57,9 @@ type Node struct {
 	RPCServer     *rpc.Server     // not optional
 	GatewayServer *gateway.Server `optional:"true"`
 
+	// block store
+	EDSStore *eds.Store `optional:"true"`
```

Comment:
This is needed for swamp testing (ability to look into EDSStore the same way we can do with header store).